lib/scenic/configuration.rb

```diff
@@ -0,0 +1,34 @@
+module Scenic
+  class Configuration
```

Comment:
It looks like it has to be `Rails.application.config.x.scenic.database` (x for extension, I guess).

I'm not sure how I feel about reaching into the Rails.application.config store versus having developers read from that store to initialize our configuration. The latter seems less magical, but I'm willing to yield to convention here.